### PR TITLE
Fixes App Bar Shadow

### LINKED ISSUE #47: Visual bug on app bar
e2cbb3c caused an issue that on some screens, the shadow looking good on the main screen doesn't look good on colored areas such as user profiles or colored tag pages.

screenshot_2018-07-25 ayarlar - flightsimtr - havacilik forumu

A solution is necessary for this **without the removal** of the shadow from the app bar as it's necessary according to Material guidelines.

### PR DESCRIPTION
**Pull request by ghost**
Fixes #47 

**Changes:**

Fixed Shadow issue occurring to basically every App Bar.

**Reviewers should focus on:**

element.style {
}
@media (min-width: 768px)
.affix .App-header {
    position: fixed;
}
`.App-header {
    padding: 8px;
    height: 52px;
    position: absolute;
    top: 0;
    left: 0;
    right: 0;
    z-index: 1000;
    box-shadow: 0px 2px 2px rgba(0,0,0,0.25);
}`

**Screenshots:**
screenshot 189

### CODE REVIEW

File: material.css [hunk 1]
```diff
@@ -12,6 +12,17 @@
...
 
+.App-header {
+    padding: 8px;
+    height: 52px;
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    z-index: 1000;
+    box-shadow: 0px 2px 2px rgba(0,0,0,0.25);
```
Comment (ardacebi) [inline material.css]: Can you decrease the shadow a bit? It looks a lot after scrolling down as it increases that time.

Comment (ghost) [reply material.css]: Let me see what I can do.

Comment (ghost) [reply material.css]: Fixed!

**Review by ardacebi — APPROVED** [review]
Does the trick, thank you.

**Pull request merged**